lambdas/src/apis/contracts/DAOListsClient.ts

```diff
@@ -0,0 +1,23 @@
+import { DAOListContract } from "decentraland-katalyst-contracts/DAOListContract";
+
+export interface DAOListClient {
+    getAllValues(): Promise<Set<string>>;
+}
+
+export class DAOListContractClient {
+
+  constructor(private readonly contract: DAOListContract) {  }
+
+  async getAllValues(): Promise<Set<string>> {
+    const count = await this.contract.getCount()
+    const values: Set<string> = new Set()
+    for (let i = 1; i <= count; i++) {
```

Comment:
Why are we starting with 1? Is this ok?

Reply:
Yes... :( 
The 0 position is a reserved value according to Nacho.

Reply:
I changed the code to use the `get`  method in the contract, it starts at 0.